releases/release-4.0.11.md

```diff
@@ -0,0 +1,165 @@
+---
+title: TiDB 4.0.11 Release Notes
+---
+
+# TiDB 4.0.11 Release Notes
+
+发版日期：2021 年 2 月 4 日
+
+TiDB 版本：4.0.11
+
+## 新功能
+
++ TiDB
+
+    - 支持 `uft8_unicode_ci` 和 `utf8mb4_unicode_ci` 排序规则 [#22558](https://github.com/pingcap/tidb/pull/22558)
+
++ TiKV
+
+    - 支持 `utf8mb4_unicode_ci` 排序规则 [#9577](https://github.com/tikv/tikv/pull/9577)
+    - 支持 `cast_year_as_time` 排序规则 [#9299](https://github.com/tikv/tikv/pull/9299)
+
++ TiFlash
+
+    - 增加排队处理 Coprocessor 任务的线程池以减小内存溢出几率，并增加配置项 `cop_pool_size` 和 `batch_cop_pool_size`，默认值为 `物理核数 * 2` [#1312](https://github.com/pingcap/tics/pull/1312)
+
+## 改进提升
+
++ TiDB
+
+    - 重排由 `outer join` 简化的 `inner join` 顺序 [#22402](https://github.com/pingcap/tidb/pull/22402)
+    - Grafana 面板支持多集群 [#22534](https://github.com/pingcap/tidb/pull/22534)
+    - 为多语句问题提供替代解决方案 [#22468](https://github.com/pingcap/tidb/pull/22468)
+    - 将慢查询监控区分为 `internal` 和 `general` 两类 [#22405](https://github.com/pingcap/tidb/pull/22405)
+    - 为 `utf8_unicode_ci` 和 `utf8mb4_unicode_ci` 排序规则增加接口 [#22099](https://github.com/pingcap/tidb/pull/22099)
+
++ TiKV
+
+    - 为 DBasS 添加 server 信息的监控指标 [#9591](https://github.com/tikv/tikv/pull/9591)
```

Comment:
typo：DBasS should be DBaaS

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/5458/commits/83b3704df63327c9ddcc75dbac057936f51eea7f.